partner_private_information_autocomplete/models/res_partner.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2024 Roger Sans <roger.sans@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    private_phone = fields.Char(
+        string="Internal Phone",
+        store=True,
+        readonly=False,
+        compute="_compute_private_phone",
+    )
+
+    private_mobile = fields.Char(
+        string="Internal Mobile",
+        store=True,
+        readonly=False,
+        compute="_compute_private_mobile",
+    )
+
+    private_email = fields.Char(
+        string="Internal Email",
+        store=True,
+        readonly=False,
+        compute="_compute_private_email",
+    )
+
+    private_street = fields.Char(
+        string="Internal Street",
+        store=True,
+        readonly=False,
+        compute="_compute_private_street",
+    )
+
+    private_street2 = fields.Char(
+        string="Internal Street2",
+        store=True,
+        readonly=False,
+        compute="_compute_private_street2",
+    )
+
+    private_zip = fields.Char(
+        string="Internal Zip",
+        change_default=True,
+        store=True,
+        readonly=False,
+        compute="_compute_private_zip",
+    )
+
+    private_city = fields.Char(
+        string="Internal City",
+        store=True,
+        readonly=False,
+        compute="_compute_private_city",
+    )
+
+    @api.depends("phone")
+    def _compute_private_phone(self):
+        self.private_phone = (
+            self.phone if not self.private_phone else self.private_phone
+        )
+
+    @api.depends("mobile")
+    def _compute_private_mobile(self):
+        self.private_mobile = (
+            self.mobile if not self.private_mobile else self.private_mobile
+        )
+
+    @api.depends("email")
+    def _compute_private_email(self):
+        self.private_email = (
+            self.email if not self.private_email else self.private_email
+        )
+
+    @api.depends("street")
+    def _compute_private_street(self):
+        self.private_street = (
+            self.street if not self.private_street else self.private_street
+        )
+
+    @api.depends("street2")
+    def _compute_private_street2(self):
+        self.private_street2 = (
+            self.street2 if not self.private_street2 else self.private_street2
+        )
+
+    @api.depends("zip")
+    def _compute_private_zip(self):
+        self.private_zip = self.zip if not self.private_zip else self.private_zip
```

Comment:
It is a multi function. You need a for loop.

Reply:
Done